backends/ebpf/bpfinclude/ebpf_registry.h

```diff
@@ -0,0 +1,133 @@
+/*
+Copyright 2018 VMware, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+ * This file defines a shared registry. It is required by the p4c-ebpf test framework
+ * and acts as an interface between the emulated control and data plane. It provides
+ * a mechanism to access shared tables by name or id (TODO: Add id map) and is
+ * intended to approximate the kernel ebpf object API as closely as possible.
+ * This library is currently not thread-safe.
+ */
+
+#ifndef BACKENDS_EBPF_BPFINCLUDE_EBPF_REGISTRY_H_
+#define BACKENDS_EBPF_BPFINCLUDE_EBPF_REGISTRY_H_
+
+#include "ebpf_map.h"
+
+#define VAR_SIZE 32  // maximum length of the table name
```

Comment:
If that's what it means, please name it MAX_TABLE_NAME_LENGTH

Reply:
done